bundle/camunda-saas-bundle/pom.xml

```diff
@@ -65,6 +65,12 @@
       <artifactId>spring-security-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.camunda.connector</groupId>
+      <artifactId>secret-provider</artifactId>
+      <version>8.8.0-SNAPSHOT</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
We usually dont reference using the version but declare the version in the parent POM dependencies.